src/core_plugins/kibana/public/management/app.html

```diff
@@ -14,8 +14,8 @@
 
       <!-- Tabs. -->
       <div data-transclude-slot="bottomRow" class="kuiLocalTabs">
-        <header ng-if="!sectionName" id="tabHeader" tabindex="0">
-          <h2 class="kuiLocalTab">
+        <header ng-if="!sectionName" id="tabHeader">
+          <h2>
```

Comment:
Also, as a general rule of thumb, we should avoid removing classes that affect the UI unless we're confident that this is a good change. In this case, this is a bad change because it introduces inconsistency between the headers of the different apps. Feel free to ping members of the design team with questions about this kind of stuff in the future.